src/index.js

```diff
@@ -1,16 +1,84 @@
 import './style.css';
 
-const populateUl = () => {
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net';
+
+// create new game
+async function getID() {
+  const response = await fetch(
+    `${baseUrl}/api/games/`,
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        name: 'Call Of Duty',
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    },
+  );
+
+  const id = await response.json();
+  return id;
+}
+
+getID().then((id) => {
+  localStorage.setItem('Game-ID', JSON.stringify(id));
+});
+
+async function getScores() {
   const list = document.querySelector('.score-board');
-  list.insertAdjacentHTML('beforeend', `
-                              <li class="score">Name : 100</li>
-                              <li class="score">Name : 20</li>
-                              <li class="score">Name : 50</li>
-                              <li class="score">Name : 78</li>
-                              <li class="score">Name : 125</li>
-                              <li class="score">Name : 77</li>
-                              <li class="score">Name : 42</li>
-                            `);
-};
-
-window.onload = populateUl();
\ No newline at end of file
+
+  list.innerHTML = '';
+
+  const response = await fetch(
+    `${baseUrl}/api/games/nBQCymqMWMPxm9purHyt/scores`,
+  );
+  const scores = await response.json();
+  return scores;
+}
```

Comment:
- [ ] Great implementation of `async` and `await` with the `fetch` method to send and receive data :+1:  , However, I can not see the data updated with the new information I added on the scoreboard after submission and making a click on the refresh button, 

![Screenshot from 2021-12-22 19-15-26](https://user-images.githubusercontent.com/57562869/147130917-97c525d3-4fb7-4bcf-bd0a-c77aec1d3e20.png)

Kindly fix the issue and make sure that the app runs as expected before submiting for the second review.